.github/scripts/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+ROOT_DIR="$(pwd)"
+FFMPEG_VERSION="6.0"
+NUM_JOBS="4"
+
+if [ $# -eq 1 ]; then
+	PACKAGE_NAME="$1"
+	FFMPEG_OPTIONS=""
+	PSXAVENC_OPTIONS=""
+elif [ $# -eq 3 ]; then
+	PACKAGE_NAME="$1"
+	FFMPEG_OPTIONS="--arch=x86 --target-os=mingw32 --cross-prefix=$2-"
+	PSXAVENC_OPTIONS="--cross-file $3"
+else
+	echo "Usage: $0 <package name> [cross prefix] [cross file]"
+	exit 0
+fi
+
+## Download FFmpeg
+
+if [ ! -d ffmpeg-$FFMPEG_VERSION ]; then
+	wget "https://ffmpeg.org/releases/ffmpeg-$FFMPEG_VERSION.tar.xz" \
+		|| exit 1
+	tar Jxf ffmpeg-$FFMPEG_VERSION.tar.xz \
+		|| exit 1
+
+	rm -f ffmpeg-$FFMPEG_VERSION.tar.xz
+fi
+
+## Build FFmpeg
+
+mkdir -p ffmpeg-build
+cd ffmpeg-build
+
+../ffmpeg-$FFMPEG_VERSION/configure \
+	--prefix="$ROOT_DIR/ffmpeg-dist" \
+	$FFMPEG_OPTIONS \
+	--enable-gpl \
+	--enable-version3 \
+	--enable-static \
+	--disable-shared \
+	--enable-small \
+	--disable-programs \
+	--disable-doc \
+	--disable-avdevice \
```

Comment:
If you want to get it even slimmer, try adding:
* `--disable-protocols --enable-protocol=file` (no non-file access)
* `--disable-iconv`

What I do is  `--disable-everything` and only allow specific decoders, as in https://github.com/WonderfulToolchain/wonderful-packages/blob/main/packages/runtime-ffmpeg/PKGBUILD#L33-L43 - but that's probably too extreme.